src/global/client/pages/login/components/Logout/Logout.tsx

```diff
@@ -13,6 +13,9 @@ import './_Logout.scss';
 
 // Behaviors and Actions
 import {
+    onLogoutRequest,
+    onLogoutResponse,
+    doLogout
 } from '../../actions';
 
 import jsonFetch, { IJSONFetch } from '../../../../api/jsonfetch'
```

Comment:
I think, we should have simpler paths like "jsonfetch" instead of "../../../..jsonfetch"; 
Otherwise, it will be painful.
I will look into that and may refactor the code later if I find a solution.